app/config/app.php

```diff
@@ -27,7 +35,7 @@
      *
      * Default value: 'Directory Lister'
      */
-    'site_title' => DI\env('SITE_TITLE', 'Directory Lister'),
+    'site_title' => DI\env('SITE_TITLE', '• Directory Lister'),
```

Comment:
The bullet has been moved here, so out of the box installations will still have a separator @PHLAK